src/org/isf/patient/service/PatientIoOperationRepositoryImpl.java

```diff
@@ -0,0 +1,113 @@
+package org.isf.patient.service;
+
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.criteria.*;
```

Comment:
No wildcard imports; enumerate them all.

Reply:
This is not resolved, @ArnaudFonzam you need to remove `import javax.persistence.criteria.*;` (that uses wildcards) and replace it with punctual import. You can use your IDE for that (in Eclipse -> "organize import" function)

Reply:
ok, the whole file has been deleted